docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
     ports:
       - "7083:8000"
       - "8083:8080"
+      - "42000-42020"
```

Comment:
I don't think this is going to get you where you want to go. There is nothing listening on those ports in the EM-API container. Also, when we add the AVL processors, they will be running in a different container.